### DISCUSSION TITLE
Red bar

### DISCUSSION THREAD

**Discussion by formigadiablo** [jgraichen/redmine_dashboard#353] [Q&A] [answered]
What exactly indicates the colour of the horizontal bar on the top?

**Comment by jgraichen** [comment, marked as answer]
The colored bar indicates how many issues are in the first columns (usually "New" or "Open"), any middle column, and in the last column ("closed"). It can serve as a simple progress indicator.

Example: Here is one issue in each section, the colored bar shows equal amount of red, yellow, and green.

![Screen Shot 2023-03-19 at 21 08 44](https://user-images.githubusercontent.com/800848/226206218-c1e7e75c-b0a9-49b0-b647-c0a88e795dcd.png)

**Comment by formigadiablo** [comment]
Thank you!
